execution/gethexec/sequencer.go

```diff
@@ -101,6 +104,12 @@ func (c *SequencerConfig) Validate() error {
 
 type SequencerConfigFetcher func() *SequencerConfig
 
+type EspressoFinalityNodeConfig struct {
+	HotShotUrl string `koanf:"hot-shot-url"`
```

Comment:
Can you confirm what we are using for the hotshot url now? I think we are using `hotshot-url` or some other things. Let's make them the same

Reply:
Yes - it is `hotshot-url`. Will use that, thank you!